airsonic-main/src/main/webapp/WEB-INF/jsp/playQueue.jsp

```diff
@@ -426,6 +454,12 @@
             }
         }
 
+        if (internetRadioEnabled) {
```

Comment:
This could be replaced with `$(".playqueue-actions").toggle(internetRadioEnabled);` I think